docs/arf.md

```diff
@@ -293,7 +293,7 @@ journey in this dynamic sector.
 This chapter describes the EUDI Wallet ecosystem as it
 is foreseen in the Commission's legislative proposal. The different
 roles in the EUDI Wallet ecosystem are described in figure 1 and
-detailed in the following sections.
+detailed in the following Sections.
```

Comment:
I do not think we should capitalize this.

Reply:
Agreed, updated in accordance.